lighthouse-core/audits/time-to-interactive.js

```diff
@@ -103,28 +108,42 @@ class TTIMetric extends Audit {
           model, artifacts.traceContents, startTime, endTime, [percentile]);
         const estLatency = latencies[0].time.toFixed(2);
         foundLatencies.push(Object.assign({}, {
-          startTime: startTime.toFixed(2),
+          startTime: startTime.toFixed(1),
           estLatency
         }));
         // console.log('At', startTime.toFixed(2), '90 percentile est latency is ~', estLatency);
         // Grab this latency and try the threshold again
         currentLatency = estLatency;
       }
+      const timeToInteractive = startTime.toFixed(1)
+
+      // Use the CDF of a log-normal distribution for scoring.
+      //   < 1200ms: score≈100
+      //   5000ms: score=50
+      //   >= 15000ms: score≈0
+      const distribution = TracingProcessor.getLogNormalDistribution(SCORING_MEDIAN,
+          SCORING_POINT_OF_DIMINISHING_RETURNS);
+      let score = 100 * distribution.computeComplementaryPercentile(timeToInteractive);
+
+      // Clamp the score to 0 <= x <= 100.
+      score = Math.min(100, score);
+      score = Math.max(0, score);
+      score = Math.round(score);
 
```

Comment:
Thanks for adding in the scoring ⭐ ⭐ ⭐ ⭐ ⭐ 

<img width="581" alt="screen shot 2016-06-22 at 12 47 07" src="https://cloud.githubusercontent.com/assets/110953/16265478/7fd24a08-3877-11e6-8ed7-e29340e3ba64.png">